### PR TITLE
HTS-207 : Use Consistent config format 

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -331,10 +330,6 @@ assets {
   url = "http://localhost:9032/assets/"
```
Comment (tim-holbrook) [inline conf/application.conf]: Pls can you make this on relative as well

Comment (tim-holbrook) [inline conf/application.conf]: Pls can you make this on relative as well

File: test/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnectorSpec.scala [hunk 2]
```diff
@@ -54,38 +44,24 @@ class EmailVerificationConnectorSpec extends UnitSpec with TestSupport with Serv
       email,
       "awrs_email_verification",
       "PT2H",
-      if (isNewApplicant) "http://localhost:7000/help-to-save/email-verified?p=" else "http://localhost:7000/help-to-save/account/email-verified?p=",
+      if (isNewApplicant) "http://localhost:7000/help-to-save/register/email-verified?p=" else "http://localhost:7000/help-to-save/account/email-verified?p=",
```
Comment (tim-holbrook) [inline test/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnectorSpec.scala]: Should the base URL and port come from config ?

File: test/uk/gov/hmrc/helptosavefrontend/connectors/IvConnectorSpec.scala [hunk 3]
```diff
@@ -36,7 +36,7 @@ class IvConnectorSpec extends TestSupport with ScalaFutures {
 
     val journeyId = JourneyId(UUID.randomUUID().toString)
 
-    val url = s"http://localhost:9938/mdtp/journey/journeyId/${journeyId.Id}"
+    val url = s"http://localhost:9948/mdtp/journey/journeyId/${journeyId.Id}"
```
Comment (tim-holbrook) [inline test/uk/gov/hmrc/helptosavefrontend/connectors/IvConnectorSpec.scala]: Should the base URL and port come from config?

File: app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala [hunk 4]
```diff
@@ -72,23 +78,23 @@ object FrontendAppConfig extends AppConfig with ServicesConfig {
...
   }
 
-  val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.create-account-url")}"
+  val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}/nsihts/createaccount"
 
-  val nsiUpdateEmailUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.update-email-url")}"
+  val nsiUpdateEmailUrl: String = s"${baseUrl("nsi")}/nsihts/updateemail"
 
-  val nsiHealthCheckUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.health-check-url")}"
+  val nsiHealthCheckUrl: String = s"${baseUrl("nsi")}/nsihts/healthcheck"
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala]: the health check and update email urls are actually always going to be `nsihts/createaccount` in reality - `nsihts/updateemail` and `nsihts/healthcheck` only exist in our stub , so these need to be read from config

File: conf/application.conf (hunk 1 above)
Comment (sureshhmrc) [reply conf/application.conf]: for some reason , it looks like its expecting full url , all other services use the same format
`https://github.tools.tax.service.gov.uk/HMRC/app-config-qa/search?utf8=%E2%9C%93&q=assets.url&type=Code`

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**